app/views/layouts/_application_content.html.erb

```diff
@@ -2,20 +2,32 @@
   <div id="content" class="container">
     <%= notifications %>
     <% unless @page_header.blank? %>
-      <div class="row form-group">
-        <h1 class="col-md-8"><%=h @page_header %></h1>
-      </div>
+        <div class="row form-group">
+          <h1 class="col-md-8"><%=h @page_header %></h1>
+        </div>
+    <% end %>
+    <% if @count_nil_hosts != nil && @count_nil_hosts > 0 %>
+        <%= alert :class => 'alert-warning', :header => '',
+                  :text => (n_('There is', 'There are', @count_nil_hosts) + ' ' +
```

Comment:
we tend to use `N_('...')` but in this case I think just `_('...')` is needed

Reply:
btw I see you didn't touch it now, but it could be perhaps improved as part of the change